src/main/java/org/parabot/core/ui/BotUI.java

```diff
@@ -143,7 +146,16 @@ public void actionPerformed(ActionEvent e) {
 
         switch (command) {
             case "Create screenshot":
-                JOptionPane.showMessageDialog(this, "We are still working on this...");
+                try {
+                    Robot robot = new Robot();
+                    Rectangle parabotScreen = new Rectangle((int) getLocation().getX(), (int) getLocation().getY(), getWidth(), getHeight());
+                    BufferedImage image = robot.createScreenCapture(parabotScreen);
+                    ImageIO.write(image, "png", new File(Directories.getScreenshotDir().getPath() + "/" + StringUtils.randomString(10) + ".png"));
+                } catch (IOException k) {
+                    k.printStackTrace();
```

Comment:
Both stacktraces have the same result, why not merge them using `IOException | AWTException k`